src/wp-includes/class-walker-nav-menu.php

```diff
@@ -39,6 +39,30 @@ class Walker_Nav_Menu extends Walker {
 		'id'     => 'db_id',
 	);
 
+	/**
+	 * The URL to the privacy policy page.
+	 *
+	 * @since 6.7.2
+	 * @var string
+	 */
+	private static $privacy_policy_url = null;
+
+	/**
+	 * Get the URL to the privacy policy page.
+	 * This method caches the URL to avoid multiple database queries and shared the URL between all instances of the class.
+	 * Single query is performed on the first call to this method.
+	 *
+	 * @since 6.7.2
+	 *
+	 * @return string The URL to the privacy policy page.
+	 */
+	public static function check_privacy_policy_url() {
```

Comment:
There is no reason for this to be public.

Also, I would rename to `get_privacy_policy_url()`